ci_framework/roles/libvirt_manager/templates/inventory.yml.j2

```diff
@@ -0,0 +1,10 @@
+{{ vm_type }}s:
+  hosts:
+{% for host in hosts %}
+    {{ vm_type }}-{{ host.item }}:
+      ansible_host: {{ (host.stdout.split())[3] | replace('/24','') }}
+      ansible_user: {{ admin_user }}
+{% if admin_user == 'core' %}
+      ansible_ssh_private_key_file: ~/.ssh/crc_key
```

Comment:
There is no mention of `crc_key` in the ci-framework. is this something coming in the new pr?

Reply:
that inventory thing is then consumed in the reproducer role - and the key is also generated/managed in that other role